pkg/controller/kogitoruntime/deployer.go

```diff
@@ -60,11 +62,34 @@ func onGetComparators(comparator compare.ResourceComparator) {
 			WithType(reflect.TypeOf(corev1.ConfigMap{})).
 			WithCustomComparator(protoBufConfigMapComparator).
 			Build())
+
+	comparator.SetComparator(
+		framework.NewComparatorBuilder().
+			WithType(reflect.TypeOf(monv1.ServiceMonitor{})).
+			WithCustomComparator(framework.CreateServiceMonitorComparator()).
+			Build())
+}
+
+func onObjectsCreate(cli *client.Client, kogitoService v1alpha1.KogitoService) (resources map[reflect.Type][]resource.KubernetesResource, lists []runtime.Object, err error) {
+	resources = make(map[reflect.Type][]resource.KubernetesResource)
+
+	resObjectList, resType, res := createProtoBufConfigMap(kogitoService)
+	lists = append(lists, resObjectList)
+	resources[resType] = []resource.KubernetesResource{res}
+
+	log.Info("going to create Prometheus service monitor")
```

Comment:
This log is printed in operator output every time the reconcile loop is called. Is it necessary?

Reply:
removed these extra debugger